tools/sim/launch_openpilot.sh

```diff
@@ -5,7 +5,13 @@ export NOBOARD="1"
 export SIMULATION="1"
 export FINGERPRINT="HONDA CIVIC 2016"
 
-export BLOCK="camerad,loggerd,encoderd"
+PROC="camerad,loggerd,encoderd"
+if [[ "$CI" ]]
+then
+  PROC="${PROC},navd,ui"
```

Comment:
why doesn't navd work? for UI, I think there's some kind of offscreen thing we can do with Qt